### ISSUE TITLE
[BUG] Pause Kodi on your remote makes wako notification not work

### ISSUE THREAD

**Issue by oathzed** [wako-app/wako#185]
So a lot of the time I'll start playing something through wako and then sometimes i'll pause it with my tv remote control, the problem is it doesn't send the callback to the wako remote so the remote doesnt say its paused. So if you try to play it again with the wako remote it just wont do anything and the wako notification is like frozen.

**Comment by JumBay** [comment]
Does this happen while the app is open or went to background after few seconds? If the app is in background for a long time and android block it nothing can be done

**Comment by oathzed** [comment]
> Does this happen while the app is open or went to background after few seconds? If the app is in background for a long time and android block it nothing can be done

Yeah I guess it only happens when I've been watching something a while and my phone has been locked for a while. It doesn't happen when my phone is open to the app or when I've been playing something for not very long
